gui/src/3D/behaviors/defaultBehavior.ts

```diff
@@ -69,8 +88,9 @@ export class DefaultBehavior implements Behavior<Mesh> {
      * Attaches the default behavior
      * @param ownerMesh The top level mesh
      * @param draggablesMeshes Descendant meshes that can be used for dragging the owner mesh
+     * @param sceneUnderstandingMeshes Meshes from the scene understanding that will be used for surface magnetism
      */
-    public attach(ownerMesh: Mesh, draggablesMeshes?: Mesh[]): void {
+    public attach(ownerMesh: Mesh, draggablesMeshes?: Mesh[], sceneUnderstandingMeshes?: AbstractMesh[]): void {

```

Comment:
is there a way to add/remove meshes from the meshes array?

Reply:
Now yes, `control.defaultBehavior.surfaceMagnetismBehavior.meshes`. It's a bit tedious, so it may be better to keep a reference of the array